mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -38,3 +38,9 @@ CLASS net/minecraft/class_2960 net/minecraft/util/Identifier
 	METHOD method_20209 isNamespaceValid (Ljava/lang/String;)Z
 		ARG 0 namespace
 	CLASS class_2961 Serializer
+		METHOD deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
+			ARG 1 json
+			ARG 3 context
+		METHOD serialize (Ljava/lang/Object;Ljava/lang/reflect/Type;Lcom/google/gson/JsonSerializationContext;)Lcom/google/gson/JsonElement;
+			ARG 1 id
+			ARG 3 context
```

Comment:
map arg 2 for these, even though they may appear red in enigma

Reply:
The middle one is `Type type1`. Enigma doesn't let me rename them to that.